nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/bulletins/ui/bulletin-board/bulletin-board-list/bulletin-board-list.component.ts

```diff
@@ -148,7 +148,7 @@ export class BulletinBoardList implements AfterViewInit {
             case 'warning':
                 return 'bulletin-warn';
```

Comment:
should this one use the `caution-color`

Reply:
No. Bulletins have a `.bulletin-error` state that is red. This is correct.